### ISSUE TITLE
Package Deprecated

### ISSUE THREAD

**Issue by mroberty8** [jsdoc3/jsdoc#1187]
Hi wrench package seems to be deprecated. a fix soon ?

```bash
npm WARN deprecated wrench@1.5.9: wrench.js is deprecated! You should check out fs-extra
 (https://github.com/jprichardson/node-fs-extra) for any operations you were using wrench for.
Thanks for all the usage over the years.
```

Thx in advance

**Comment by eden** [comment]
Related: https://github.com/jsdoc3/jsdoc/pull/1186

**Comment by popomore** [comment]
use mkdirp instead?

**Comment by superclarkk** [comment]
+1 pleeeease

**Comment by joekukish** [comment]
+1

**Comment by sulliwane** [comment]
+1

**Comment by hegemonic** [comment]
Fixed by #1186.
